Dropdown/Dropdown.js

```diff
@@ -46,6 +46,7 @@ import componentCss from './Dropdown.module.less';
 
 const oppositeDirection = {left: 'right', right: 'left', up: 'down', down: 'up'};
 const ContainerDiv = SpotlightContainerDecorator({enterTo: 'last-focused'}, 'div');
+const Button = Skinnable(ButtonBase);
```

Comment:
You also removed other decorator . 

Pure,
	IconButtonDecorator,
	TooltipDecorator({tooltipDestinationProp: 'decoration'}),
	UiButtonDecorator,
	Spottable

Is it intended? @adrian-cocoara-lgp @stanca-pop-lgp @daniel-stoian-lgp 
Because of this, I can't reach the dropdown with the 5way key in silicon.

I think we need to fix around 291 Line. But before that, DrowDown doesn't sync up with sandstone?

Reply:
You are right. I added the decorators below. IconButtonDecorator was not added since there is use case for a button with icon for dropdown.

Pure,
UiButtonDecorator,
TooltipDecorator({tooltipDestinationProp: 'decoration'}),
Spottable,
Skinnable

Reply:
we will analyze the sync of Dropdown very soon! 👍